tuf/src/client.rs

```diff
@@ -1016,6 +1000,30 @@ where
     }
 }
 
+// For some reason clippy thinks we can elide the lifetime, but rust needs it so
+// since the fetch future can hold a reference to repo items.
```

Comment:
nit: "so since the fetch future can hold"

Reply:
fixed